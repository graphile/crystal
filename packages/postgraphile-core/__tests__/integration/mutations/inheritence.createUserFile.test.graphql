#> schemas: "inheritence"

mutation {
  createUser(input: {
    user: {
      id: 1
      name: "Bobby Tables"
    }
  }) {
    user {
      nodeId
      id
      name
    }
  }
  createUserFile(input: {userFile: {filename: "foo.txt", userId: 1}}) {
    userFile {
      id
      filename
      userByUserId {
        id
        name
      }
    }
  }
}