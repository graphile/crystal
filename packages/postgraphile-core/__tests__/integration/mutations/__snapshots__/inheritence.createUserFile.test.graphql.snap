// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "createUser": Object {
      "user": Object {
        "id": 1,
        "name": "Bobby Tables",
        "nodeId": "WyJ1c2VycyIsMV0=",
      },
    },
    "createUserFile": Object {
      "userFile": Object {
        "filename": "foo.txt",
        "id": 1,
        "userByUserId": Object {
          "id": 1,
          "name": "Bobby Tables",
        },
      },
    },
  },
}
`;
